packer/builder/azure/config.go

```diff
@@ -44,6 +44,7 @@ type Config struct {
 	tmpServiceName   string
 	tmpContainerName string
 	userImageName    string
+        SSHPrivateIp     bool   `mapstructure:"ssh_private_ip"`
```

Comment:
I rebased my branch before I pushed, but I recall originally using `use_private_ip`.  For some reason, I had a problem in that my config was never picked up, and was always empty.  That's why I ended up changing to `SSHPrivateIp`.

It's highly likely I was doing something wrong.  I'll try changing the value around and re-testing at my next opportunity.